src/huggingface/forte/huggingface/token_classification.py

```diff
@@ -0,0 +1,204 @@
+        Returns:
+            result_types: list of merged entity type.
+            result_indices: list of (start, end) index for the merged entities.
+
+        """
+        indices = [(token["start"], token["end"]) for token in tokens]
+
+        tags: List[str] = []
+        types: List[str] = []
+
+        for token in tokens:
+            if re.match("^[B|I]-.*", token["entity"]):
+                tag, type = token["entity"].split("-")
+            elif token["entity"] in ["B", "I"]:
+                tag, type = token["entity"], ""
+            else:
+                raise ValueError("cannot process the BIO pattern.")
+            tags.append(tag)
+            types.append(type)
+
+        result_types, result_indices = bio_merge(tags, types, indices)
+        return result_types, result_indices
+
+    @classmethod
+    def default_configs(cls):
+        r"""This defines a basic config structure for TokenClassification.
+
+        Following are the keys for this dictionary:
+            - `entry_type`: defines which entry type in the input pack to make
+              prediction on. The default makes prediction on each `Sentence`
+              in the input pack.
+            - `output_entry_type`: defines which entry type in the input pack
+              that the prediction should be saved as output. The default
+              saves prediction on `Token` in the input pack.
+            - `task`: defines the nlp task, for example, "ner", "pos", "ws".
+            - `attribute_name`: defines which attribute of the
+              `output_entry_type` in the input pack to save prediction to.
+              The default saves prediction to the `ner` attribute for each
+              `output_entry_type` in the input pack.
+            - `tagging_scheme`: defines the result merged tagging_scheme,
+              for example, "no-merge" to output the original result,
+              "bio-merge" to merge the BIO tagging format result.
+              The default is "no-merge".
+            - `model_name`: language model, default is `"dslim/bert-base-NER"`.
+              The wrapper supports HuggingFace models with pipeline tag of
+              `ner`.
+            - `tokenizer`: language model for tokenization, default is
+              `"dslim/bert-base-NER"`.
+            - `framework`: The framework of the model, should be `"pt"` or
+              `"tf"`. This information should be obtained from Huggingface
+              model hub.
+            - `cuda_device`: Device ordinal for CPU/GPU supports. Setting
+              this to -1 will leverage CPU, a positive will run the model
+              on the associated CUDA device id. Default is -1.
+
+        Returns: A dictionary with the default config for this processor.
+        """
+        config = super().default_configs()
+        config.update(
+            {
+                "entry_type": "ft.onto.base_ontology.Sentence",
+                "output_entry_type": "ft.onto.base_ontology.Token",
```

Comment:
If we want to use NER as default, let's not use `Token.ner` to store them, let's make the default to be `EntityMention`, and `tagging-schem` to be `bio-merge`.